frontend/src/tests/pages/UCSBOrganization/UCSBOrganizationCreatePage.test.js

```diff
@@ -0,0 +1,117 @@
+import { render, screen, fireEvent, waitFor } from "@testing-library/react";
+import UCSBOrganizationCreatePage from "main/pages/UCSBOrganization/UCSBOrganizationCreatePage";
+import { QueryClient, QueryClientProvider } from "react-query";
+import { MemoryRouter } from "react-router-dom";
+
+import { apiCurrentUserFixtures } from "fixtures/currentUserFixtures";
+import { systemInfoFixtures } from "fixtures/systemInfoFixtures";
+import axios from "axios";
+import AxiosMockAdapter from "axios-mock-adapter";
+
+const mockToast = jest.fn();
+jest.mock('react-toastify', () => {
+    const originalModule = jest.requireActual('react-toastify');
+    return {
+        __esModule: true,
+        ...originalModule,
+        toast: (x) => mockToast(x)
+    };
+});
+
+const mockNavigate = jest.fn();
+jest.mock('react-router-dom', () => {
+    const originalModule = jest.requireActual('react-router-dom');
+    return {
+        __esModule: true,
+        ...originalModule,
+        Navigate: (x) => { mockNavigate(x); return null; }
+    };
+});
+
+describe("UCSBOrganizationCreatePage tests", () => {
+
+    const axiosMock = new AxiosMockAdapter(axios);
+
+    beforeEach(() => {
+        jest.clearAllMocks();
+        axiosMock.reset();
+        axiosMock.resetHistory();
+        axiosMock.onGet("/api/currentUser").reply(200, apiCurrentUserFixtures.userOnly);
+        axiosMock.onGet("/api/systemInfo").reply(200, systemInfoFixtures.showingNeither);
+    });
+
+    const queryClient = new QueryClient();
+    test("renders without crashing", () => {
+        render(
+            <QueryClientProvider client={queryClient}>
+                <MemoryRouter>
+                    <UCSBOrganizationCreatePage />
+                </MemoryRouter>
+            </QueryClientProvider>
+        );
+    });
+
+
+    test("on submit, makes request to backend, and redirects to /ucsborganization", async () => {
+
+        const queryClient = new QueryClient();
+        const restaurant = {
+            orgCode: "ZPR",
+            orgTranslationShort: "ZETA PHI RHO",
+            orgTranslation: "ZETA PHI RHO",
+            inactive: "false"
+        };
+
+        axiosMock.onPost("/api/ucsborganization/post").reply(202, restaurant);
```

Comment:
this should be plural "/ucsborganizations/post"

Reply:
my bad, will fix ty!

Reply:
> my bad, will fix ty!

ofc :)